docs/kb/function-tutorial-intro.md

```diff
@@ -0,0 +1,239 @@
+
+Next, install Docker with Chocolatey from a PowerShell window in Administrator mode.
+
+```
+choco install docker
+```
+
+Now login using the Docker account you just signed up with (replace dduck with your own account info below).  Open a Terminal window and submit:
+```
+docker login --username dduck
+```
+
+### Install and Configure kubectl
+
+kubectl is the command line utility used to access a Kubernetes cluster.  With kubectl, you can deploy the VEBA function in the form of Kubernetes secrets and pods, check logs, and troubleshoot the health of the Kubernetes cluster and VEBA application.  The kubectl command line utility determines which Kubernetes cluster to access and which certificate to use for authentication by reading a config file.  The default config file resides at: ~/.kube/config on the VEBA appliance.
+
+Install kubectl with Chocolately from a PowerShell window in Administrator mode:
+
+```
+choco install kubernetes-cli
+kubectl version
+```
+
+Next we will copy the .kube config file from the VEBA appliance to the local workstation.  You will need to know the IP address and root password of the deployed VEBA appliance and also verify that ssh has been enabled.  These parameters will have been set during the VEBA appliance deployment.  Once the config file is copied locally, we will export it so that it becomes the default kubectl config file.  You can switch which Kubernetes cluster kubectl points at by changing the "Env:KUBECONFIG" file link.  This example creates a "veba" folder to store the config file in and uses scp to copy the file from the VEBA appliance.  SCP is a secure copy utility and was installed as part of the Chocolately install of git earlier.  Replace "VEBA IP" below with the VEBA appliance IP address.  You will be prompted for the VEBA appliance root password.
+
+
+```
+cd ~
+mkdir veba
+cd veba
+scp 'root@<VEBA IP>:/root/.kube/config' .
+Env:KUBECONFIG="$HOME\veba\config"
+```
+The above sample code is _**one**_ way of configuring the Kubernetes config file to use for access to the cluster.  How to navigate between multiple configs is documented here: [Kubernetes configure access](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
+
+
+Check that everything is working.
+- kubectl config view: returns the currently loaded config
+- kubectl cluster-info: returns info from the Kubernetes cluster (within the VEBA appliance)
+- kubectl get ns: returns the namespaces from the Kubernetes cluster
+```
+kubectl config view
+kubectl cluster-info
+kubectl get ns
+```
+
+You should see something similar to the below image:
+
+<img src="./img/tutorial-kubectl-success1.png" width="100%" align="center" class="border m-1 p-1"/>
```

Comment:
nit: these could have been a console output like you use above instead of a screenshot.

Reply:
I've removed screenshots and replaced with console output in all cases but a few where I wanted to highlight something and the screenshot displayed things more clearly.